PostMetabox.php

```diff
@@ -292,7 +292,8 @@ protected function save( $post_id ) {
 			);
 			update_post_meta( $post_id, '_error_data_' . $this->id, $error_data );
 
-			$location = add_query_arg( 'message', 1, get_edit_post_link( $post_id, 'url' ) );
+			$location = esc_url_raw( add_query_arg( 'message', 1, get_edit_post_link( $post_id, 'url' ) ) );
+
 			wp_redirect( apply_filters( 'redirect_post_location', $location, $post_id ) );
```

Comment:
Yeah. I agree, but I also don't see WP escaping data from most of their other filters as a general rule.